fix typos v3.1 (#24)
mulesoft/docs-healthcare-toolkit: modules/ROOT/pages/hl7-edi.adoc

If you just want to delete some segments from the message structure, or change some segment requirements from mandatory to optional (or vice versa), you can easily make the change in your copy of the schema and use the modified version in your application.

> You can also add segments that are not present in the original message structure definition. For this we recommend you first remove the position values from all the existing segment and group definition lines in the schema, since otherwise you need to renumber everything following an added segment. If you remove the explicit position numbers. segments and groups are assigned position numbers sequentially, and for most purposes these numbers are not seen by HL7 EDI Connector 3.1.0 applications.

Reviewer: ```suggestion
You can also add segments that are not present in the original message structure definition. For this, it is best if you first remove the position values from all the existing segment and group definition lines in the schema, otherwise, you must renumber everything following an added segment. If you remove the explicit position numbers, segments and groups are assigned position numbers sequentially, and for most purposes, these numbers are not seen by HL7 EDI Connector 3.1.0 applications.
```

  ...
----

Unused components of a segment cannot just be dropped from the segment definition (unless they're at the end of the segment), so the simplification tool just substitutes a "varies" data type for the component and marks it with Usage: U for Unused. The repetition count for the "varies" remains the same as for the original component in this case, but it will not show up in the DataSense view of the data you see in Data Weave.

> When the schema simplification tool checks which data is present in the messages it handles each occurrence of a composite in context, so different usages of the same composite may have different values present. When this happens the composite is defined more than once, with different identifiers. The CE composite in the above example shows this.

Reviewer 2: ```suggestion
When the schema simplification tool checks which data is present in the messages, it handles each occurrence of a composite in context, so different usages of the same composite may have different values present. When this happens, the composite is defined more than once with different identifiers. The CE composite in the above example shows this.
```
